quantum/process_keycode/process_tap_term_keys.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 Vladislav Kucheriavykh
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+#include "action_tapping.h"
```

Comment:
```suggestion
#include <stdbool.h>
#include "action.h"
```

Reply:
I'm getting compile errors when using https://github.com/qmk/qmk_firmware/pull/11036/commits/5c52e1ec10566ee67a3ae8e7127d7ccb4f459913 saying the `tapping_term` variable and all the tap term keycodes are undeclared. So, I'm probably gonna revert this suggestion.

Reply:
Its just that `process_tap_term_keys.c` needs further includes.